Security/Guidelines/OpenSSH.mediawiki

```diff
@@ -433,7 +446,9 @@ Key material identifies the cryptographic secrets that compose a key. All key ma
 This includes:
 * OpenSSH server keys (<code>/etc/ssh/ssh_host_*key</code>)
 * Client keys (<code>~/.ssh/id_{rsa,dsa,ecdsa,ed25519}</code> and <code>~/.ssh/identity</code>).
-* <code>/etc/ssh/moduli</code> also contains prime numbers and generators for use in the Diffie-Hellman key exchange and must be handled like key material.
+
+This do not include:
```

Comment:
I have left it to make it clear this file had not been forgotten and to make it clear the previous version was incorrect. However, I will not oppose very hard its removal if some more people ask for it.